docs/en/development/build_osx.md

```diff
@@ -33,11 +33,23 @@ For the latest stable version, switch to the `stable` branch.
 ```bash
 $ mkdir build
 $ cd build
-$ cmake .. -DCMAKE_CXX_COMPILER=`which g++-8` -DCMAKE_C_COMPILER=`which gcc-8`
+$ cmake .. -DCMAKE_CXX_COMPILER=`which g++` -DCMAKE_C_COMPILER=`which gcc`
```

Comment:
Current version of this doc  in `master` has ``` -DCMAKE_CXX_COMPILER=`which clang++` -DCMAKE_C_COMPILER=`which clang` ``` here. So probably this line no longer requires any changes. 

On Mac OS, `gcc` binary is a shortcut to clang, but I don't know if it has any differences when invoked as `gcc` or `clang`. But error that had been fixed in this PR occurs both when using `gcc` and when using `clang`.